jplag.cli/src/main/java/de/jplag/CLI.java

```diff
@@ -36,7 +37,7 @@ public final class CLI {
 
     private static final Logger logger = LoggerFactory.getLogger(CLI.class);
 
-    private static final Random RANDOM = new Random();
+    private static final Random RANDOM = new SecureRandom();
```

Comment:
Is this really necessary? We only use the random values for the gimmick of providing different short descriptions. Whether that randomness there is predictable or not should not be important.

Reply:
Not necessary, but there should not be a disadvantage here ( otherwise we have a sonarlint issue :D )